except_star.md

```diff
@@ -75,17 +75,18 @@ unwinds. Several real world use cases are listed below.
 ## Rationale
 
 Grouping several exceptions together can be done without changes to the
-language, simply by creating a container exception type. Trio is an example of
-a library that has made use of this technique in its `MultiError` type
-[reference to Trio MultiError]. However, such approaches require calling code
-to catch the container exception type, and then inspect it to determine the
-types of errors that had occurred, extract the ones it wants to handle and
-reraise the rest.
+language, simply by creating a container exception type.
+[Trio](https://trio.readthedocs.io/en/stable/) is an example of a library that
+has made use of this technique in its
+[`MultiError` type](https://trio.readthedocs.io/en/stable/reference-core.html#trio.MultiError).
+However, such approaches require calling code to catch the container exception
+type, and then inspect it to determine the types of errors that had occurred,
+extract the ones it wants to handle and reraise the rest.
```

Comment:
Perhaps we can add a link to show that "... require calling code to catch the container exception reraise the rest. type, and then inspect it to determine the types of errors ..." is very cumbersome and unintuitive.

Also, having custom multi error implementations in multiple packages makes it very hard to use them together in one project.

Reply:
Feel free to make a PR with such changes. I'm not editing now.